scripts/ping_forever.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+#
+# The script is expected to run on admin server, takes IP as only argument.
+#
+# This is used for pinging given IP indefinitely (more specifically,
+# until interrupted by creating /var/lib/crowbar/stop_pinging.$IP file).
+#
+# Originally intended for testing the instances accessibility during the Cloud upgrade.
+
+trap '' 1
+
+fip=$1
+LOGFILE=/var/log/ping_instance.$fip.out
+failed=0
+date -Iseconds > $LOGFILE
+echo "Starting the ping of $fip" >> $LOGFILE
+
+while [ 1 = 1 ]; do
+    if [ -e /var/lib/crowbar/stop_pinging.$fip ]; then
+        echo "Stopping the ping of $fip !" >> $LOGFILE
+        exit 0
+    fi
+    if ping $fip -W 1 -w 1 > /dev/null ; then
```

Comment:
might return in 0 seconds instead of 1 seconds, if there is a router in between (which is the case for admin->floating)
```
> time ping -W 1 -w 1 10.162.191.240
PING 10.162.191.240 (10.162.191.240) 56(84) bytes of data.
From 10.160.255.253 icmp_seq=1 Destination Host Unreachable

--- 10.162.191.240 ping statistics ---
1 packets transmitted, 0 received, +1 errors, 100% packet loss, time 0ms


real    0m0.002s
user    0m0.001s
sys     0m0.000s
```